### PR TITLE
fix(application): call the documented /v1/info route

### PR DESCRIPTION
**Pull request by jpennin5**
## Description

Part of #108 (item B5).

**Problem:** `get_application_info` requests
`/integration/v1/application/info`, which no controller serves, so the
tool 404s everywhere.

**Root cause + fix:** the documented route is `/v1/info` and its
documented response is a single key, `{"applicationVersion": "..."}`. The
`version`/`build`/`deploymentType`/`capabilities`/`systemInfo` shape the
tool previously parsed came from nowhere real — on the correct path every
one of those keys would still have been `None`. The tool now reports the
documented key as `application_version` and passes any additional keys a
controller chooses to send through verbatim rather than dropping them
through an allowlist.

**Live verification (Network 10.5.67):** returns the controller's real
application version.

## Type of Change

- [x] Bug fix (non-breaking change that fixes an issue)

The output shape changes, but the old route never returned anything, so
nothing that worked can break.

## Testing

- Tests cover the documented single-key response, extra-key passthrough,
  and the unauthenticated path.
- `pytest tests/unit/` passes; pre-commit hooks pass for all files this
  PR touches.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (CHANGELOG.md)
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published (none — stands alone)

## AI Assistance

This PR was created with assistance from Claude Code.

**Human Review Status:** ✅ Reviewed and approved by @jpennin5
**Testing:** Full unit suite passing; route verified against a live
Network 10.5.67 controller.
**Security Review:** Completed — no identifying values; fixtures are
synthetic.

### CODE REVIEW

File: tests/unit/tools/test_application_tools.py [hunk 1]
```diff
@@ -20,151 +20,83 @@ def mock_settings():
     settings.local_host = "192.168.2.1"
     settings.local_port = 443
     settings.local_verify_ssl = False
+    settings.get_integration_path = MagicMock(side_effect=lambda x: f"/integration/v1/{x}")
```
[the remark this answers is not in the document]
Comment (jpennin5) [reply tests/unit/tools/test_application_tools.py]: Fixed in 0d8605a — the fixture now stubs the real local-mode prefix `/proxy/network/integration/v1/...` and the assertion follows.

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - **`list_dpi_applications` on Integration v1 (issue #108)**: `GET /integration/v1/dpi/applications` returns only `{"id": 3, "name": "ICQ"}`, but `DPIApplication` required `category_id` and typed `id` as a string, so every response raised `ValidationError` and the tool was unusable in local API mode. `category_id` is now optional, `id` accepts the numeric form the Integration v1 route sends, and `category_id`/`category_name` also accept the API's camelCase spellings.
 - **`WEB` firewall matching target (issue #106)**: `MatchingTarget` was missing the `WEB` member that the UniFi controller returns for web-category rules, so any policy using it failed model validation. Because `list_firewall_policies` validates the whole payload, a single `WEB` rule made the entire policy list unavailable rather than just that one entry.
+- **`get_application_info` (issue #108, item B5)**: requested `/integration/v1/application/info`, which no controller serves, so the tool 404'd everywhere. Now calls the documented `/v1/info` route and reports its `applicationVersion` as `application_version`, passing any additional keys a controller sends through verbatim.
```
[the remark this answers is not in the document]
Comment (jpennin5) [reply CHANGELOG.md]: Reworded in 0d8605a: the entry now says the prefix varies by API mode (`/v1` on Cloud V1, `/proxy/network/integration/v1` on local gateways) instead of implying a single literal path.

File: src/tools/application.py [hunk 3]
```diff
@@ -27,8 +27,12 @@ async def get_application_info(settings: Settings) -> dict:
         if not client.is_authenticated:
             await client.authenticate()
 
-        # Get application info
-        response = await client.get("/integration/v1/application/info")
+        # The documented route is /v1/info; /application/info does not exist
+        # and 404s on every controller. The documented response carries a
+        # single key, applicationVersion — the version/build/deploymentType/
+        # capabilities/systemInfo keys this tool used to report came from
+        # nowhere real and were always None.
```
[the remark this answers is not in the document]
Comment (jpennin5) [reply src/tools/application.py]: Reworded in 0d8605a — the comment now describes the documented resource with its mode-dependent prefix.

**Comment by enuno** [comment]
### Summary
Request changes. The PR is not yet merge-ready because the current status checks are not clean.

### Critical findings
- Merge state: \ / \.
- Failing check(s): Mermaid Diagram Sync Assistant.
- The changed tooling surface should be revalidated after the failing check is resolved.

### Suggestions
1. Resolve the failing status check(s) and rerun CI.
2. Reconfirm the changed tooling behavior against the intended controller/fixture shapes.
3. Ensure regression tests cover the original failure mode and edge cases introduced here.
4. Update docs if any request/response or error semantics changed.

### Verification checklist for the author
- [ ] Required checks are passing
- [ ] Regression tests cover the changed path
- [ ] Lint / format / type checks pass locally
- [ ] Documentation is updated where user-facing behavior changed
- [ ] No secrets, credentials, or controller-identifying data were introduced

**Comment by jpennin5** [comment]
The failing check is the `Mermaid Diagram Sync Assistant` GitHub App, not a workflow in this repository. Its check-run output on this PR reads: "Webhook Processing Failed — An unexpected error occurred while processing the webhook. Please try pushing a new commit or check your repository configuration." That processing happens on the app's side; pushing new commits to these PRs has re-triggered the check with the same result, so it is not reachable from a fork.

This is the loop documented in #112: the app has errored on every fork PR since late June, fork CI runs sit at `action_required` awaiting maintainer approval, and the daily automated review then reads the app error as a failing required check and requests changes.

Every check that can conclude on a fork PR is green here (both Socket Security checks). The remaining steps are repo-side — approving the CI runs and either repairing/removing the app integration or unrequiring its check. #117 merged with the identical check state, so the merge path exists. Happy to rebase or adjust anything code-side the moment CI can actually run.
